app/src/main/java/com/sun/uefascore/screen/fixtures/FixturesFragment.kt

```diff
@@ -17,7 +27,23 @@ class FixturesFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_fixtures, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initPresenter()
+    }
+
+    @SuppressLint("SimpleDateFormat")
```

Comment:
cáin này để làm gì e?

Reply:
Với ở dưới là do em để fotmart day nên nó đề xuất